src/spec/sync/product-sync.spec.coffee

```diff
@@ -209,3 +209,102 @@ describe 'ProductSync', ->
         ]
         version: oldProduct.version
       expect(update).toEqual expected_update
+
+    describe 'actionsMapPriceCustom', ->
+      ieNew =
```

Comment:
Something seems wrong here.
- Why are we passing variants and not products?
- The structure of the variant is also wrong. The custom price object is should be in the `price` and not directly in the variant.
I think this is how it should look:
```coffee
newProduct =
  id: '123',
  version: 1,
  variants: [{
    sku: 'abc',
    price: {
      id: '',
      value: {
        currenceyCode: 'EUR',
        centAmount: 4200
      },
      custom: {
        type: {
          typeId: 'type',
          id: '123'
        },
        fields: {
          nac: 'ho'
        }
      }
    }
  }]
```
The same applies for the next couple of tests.